data/publiccloud/aitl/shared_gallery.json

```diff
@@ -0,0 +1,65 @@
+{
+    "location": "westus3",
+    "properties": {
+        "jobTemplateInstance": {
+            "templateTags": [],
+            "selections": [
+                {
+                    "caseName": [
+                        "verify_connection_status",

```

Comment:
this should not block merge just question for future improvements ...

Reply:
There is not option to run whatever (at the time of writing). A test manifest must be specified and it has some parameters. They have added 2 new tests since I wrote this, so your gut feeling is correct :)  

As a side note - we want to keep track if the tests pass or not, and adding new tests without our knowledge (because MS updated something) can be annoying to debug.

Reply:
It might make sense to import the tests based on Tiers instead of by caseName. Looking into that.

Reply:
After discussing with our contact, it does in fact make sense to use Tiers https://mslisa.readthedocs.io/en/main/run_test/microsoft_tests.html#test-tier instead of specific caseNames. However, our initial sample cases were ~40 tests while T2 for example are 180+. So I would like to try using T1 for the time being and we can later either cherry-pick or increase/decrease the Tier.